src/ptex/PtexExports.h

```diff
@@ -54,7 +54,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
 #       if defined(PTEX_EXPORTS)
 #           define PTEXAPI __attribute__((visibility("default")))
 #       else
-#           define PTEXAPI
+#           define PTEXAPI ____declspec(dllimport)
```

Comment:
Extra underscores?
Also, I don't think that __declspec(dllimport) is understood by clang or gcc. Maybe by Intel?

Reply:
oh boy that was intended for the PTEX_PLATFORM_WINDOWS section, like it originally was. That's been fixed. We'll be adding CI pipelines for Windows to guarantee this going forward.

Reply:
Thanks! Dunno about Windows, my failure was on Linux. How did it pass CI?

Reply:
OH. I see why. Your CI only builds and runs Ptex itself, so the only path it takes through this header is the "export" branch. Your CI doesn't try to build in the way a downstream project would, consuming Ptex.